src/settings-ui/Settings.UI/OOBE/ViewModel/OobePowerToysModule.cs

```diff
@@ -41,7 +41,7 @@ public void LogOpeningSettingsEvent()
 
         public void LogRunningModuleEvent()
         {
-            PowerToysTelemetry.Log.WriteEvent(new OobeModuleRunEvent() { ModuleName = this.ModuleName });
+            PowerToysTelemetry.Log.WriteEvent(new TrayFlyoutModuleRunEvent() { ModuleName = this.ModuleName });
```

Comment:
this change looks wrong

Reply:
You're very right! Just fixed it. I used that event as a basis and then changed the wrong one. 🤦  It should be good now.